debug-quick-look.php

```diff
@@ -142,6 +142,7 @@ public function process_debug_type() {
 
 		// Create some basic CSS.
 		$style  = '
+		body#error-page { max-width: 1200px; }
```

Comment:
Sure man, you/I can definitely change that. i set it to 1200 max because beyond that it felt unreadable to me. Here is 80% on a 2560px wide screen: http://b.ustin.co/6SCKDh